components/gitpod-cli/cmd/rebuild.go

```diff
@@ -256,12 +256,15 @@ func runRebuild(ctx context.Context, supervisorClient *supervisor.SupervisorClie
 		Target     string
 		Source     string
 		Permission os.FileMode
+		Optional   bool
 	}
 
 	prepareFS := []mnte{
 		{Source: "/workspace"},
```

Comment:
we should ensure that we are using gitpod user while creating a user, otherwise it is going to break restarts

Reply:
Ok, try to test, first start a workpace just with browser, run gp validate, then restart with desktop or intellij, and run gp validate again. It broke for me for some reasons for some folders under workspace. They were created for root, so intellij could not start.